Kaguya/Kaguya/Discord/Commands/Reference/Hex.cs

```diff
@@ -0,0 +1,41 @@
+using Discord.Commands;
+using Kaguya.Internal.Attributes;
+using Kaguya.Internal.Enums;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using System;
+
+namespace Kaguya.Discord.Commands.Reference
+{
+	[Module(CommandModule.Reference)]
+	[Group("hex")]
+	[Alias("hx")]
+	public class Hex : KaguyaBase<Hex>
+	{
+		private readonly ILogger<Hex> _logger;
+
+		public Hex(ILogger<Hex> logger) : base(logger)
+		{
+			_logger = logger;
+		}
+
+		[Command]
+		[Summary("Takes a hexadecimal color value and outputs an embed with that color.")]
+		[Remarks("<hex value>")]
```

Comment:
"Hex value" is a bit of a loosely defined term for the average user, some might say #FFFFFF or 0xFFFFFF -- definitely want an example here.

`[Example("FFFFFF")]`, etc.